src/main/resources/application-dev.properties

```diff
@@ -29,10 +29,7 @@ et3Notification.template.id=${ET3_NOTIFICATION_TEMPLATE_ID:1234}
 
 queue.create-updates.queue-name = ${CREATE_UPDATES_QUEUE_NAME}
 queue.create-updates.send.connection-string = ${CREATE_UPDATES_QUEUE_SEND_CONNECTION_STRING}
-queue.create-updates.queue-name = create-updates
+#queue.create-updates.queue-name = create-updates
```

Comment:
Did you mean to commit these changes?

Reply:
Delete the commented line, perhaps line 30 could be 
`${CREATE_UPDATES_QUEUE_NAME:create-updates}`

Reply:
fixed